lib/Doctrine/Common/DataFixtures/Executor/Executor.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Common\DataFixtures\Executor;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+
+interface Executor
```

Comment:
Does this interface still have a purpose?

Reply:
It's new, and I thought it could be useful in case there is a need for external packages to type hint, against something, but maybe introduce it is premature? We might want to rethink the `$append` boolean part.

Reply:
I see, we need it because the new executor does not extend `AbstractExecutor` anymore. 😕